v2/cn/events.md

```diff
@@ -0,0 +1,15 @@
+<!---
+    @title         Events
+    @creator       Yichun Zhang
+    @modifier      Yichun Zhang
+    @created       2016-11-28 15:58 GMT
+    @changes       1
```

Comment:
We should not specify the `@creator`, `@modifier`, `@created`, and `@changes` metadata explicitly here since the toolchain can automatically derive such info from the git log history when these fields are absent, which is what we want.